### ISSUE TITLE
Table linting violations not being detected

### ISSUE THREAD

**Issue by thompson-tomo** [igorshubovych/markdownlint-cli#576]
The following tables are not being flagged based on MD060 when is set to the default value

```
| Name     | Instrument Type | Unit (UCUM) | Description    | Stability | Entity Associations |
| -------- | --------------- | ----------- | -------------- | --------- | ------ |
| `cicd.pipeline.run.duration` | Histogram | `s` | Duration of a pipeline run grouped by pipeline, state and result. | ![Development](https://img.shields.io/badge/-development-blue) | [`cicd.pipeline`](/docs/registry/entities/cicd.md#cicd-pipeline) |

```

It uses aligned for the header text & seperator line. Body uses compact.

```
| Key | Stability | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Value Type | Description | Example Values |
|---|---|---|---|---|---|
| [`cicd.pipeline.name`](/docs/registry/attributes/cicd.md) | ![Development](https://img.shields.io/badge/-development-blue) | `Required` | string | The human readable name of the pipeline within a CI/CD system. | `Build and Test`; `Lint`; `Deploy Go Project`; `deploy_to_environment` |
| [`cicd.pipeline.run.state`](/docs/registry/attributes/cicd.md) | ![Development](https://img.shields.io/badge/-development-blue) | `Required` | string | The pipeline run goes through these states during its lifecycle. | `pending`; `executing`; `finalizing` |

```

It uses compact for the header text & Body. seperator line uses tight.

```

| Value  | Description | Stability |
|---|---|---|
| `_OTHER` | A fallback error value to be used when the instrumentation doesn't define a custom value. | ![Stable](https://img.shields.io/badge/-stable-lightgreen) |


```

It uses aligned for the header text, Body uses compact and seperator line uses tight.

**Comment by thompson-tomo** [comment]
Ah ok, do we know when it will be supported? It would be good if the readme indicated the supported rules as currently the readme gives the impression that it is implemented.

**Comment by DavidAnson** [comment]
v0.46.0
